tock/tock/static/sass/_header.scss

```diff
@@ -0,0 +1,101 @@
+// To mimic behavior of primary nav dropdown
```

Comment:
Jen's design called for a dropdown (the "Admin" dropdown) in the upper part of the header. Well, turns out the USWDS header _freaks out_ when you try to use a dropdown there, so I had to pull in the styles for the dropdown that exist in the main part of the nav. I don't love this and it feels fragile to have to replicate these styles, but I don't really know that we've got an alternative that doesn't involve redesigning the nav.

Reply:
If you really dislike it, let's maybe log an issue for a designer to rethink it?